utils/keys.go

```diff
@@ -103,6 +103,9 @@ func ImportKeys(from io.Reader, to []Importer) error {
 		toWrite []byte
 	)
 	for block, rest := pem.Decode(data); block != nil; block, rest = pem.Decode(rest) {
+		if role != "" {
```

Comment:
I think this conditional logic should be slightly different to match the usage comment - we should check if `block.Headers["role"]` exists, and set it to `role` if it's empty.

Also, I think this logic should combine with the pathing logic below since to-be-imported delegation keys may not have a `path` PEM header.  We can check if we have a provided role and if it's a delegation we can write it to `private/tuf_keys/<KEY_ID>.key`.  If we follow @cyli's suggestion for GUN headers we can include that in the path we construct for non-delegation keys as well.